script/cadence/scripts/get-balances.cdc

```diff
@@ -16,14 +18,14 @@ access(all) struct AccountBalances {
 
 access(all) fun main(addr: Address): AccountBalances {
     let acct = getAccount(addr)
-    let balanceRef = acct.capabilities.borrow<&FlowToken.Vault}>(/public/flowTokenBalance)
+    let balanceRef = acct.capabilities.borrow<&FlowToken.Vault>(/public/flowTokenBalance)!
     var is_proxy = false
     var proxy_balance = 0.0
-    let ref = acct.capabilities.borrow<&{FlowColdStorageProxy.Vault}>(FlowColdStorageProxy.VaultCapabilityPublicPath)
-    if let vault = ref {
-        is_proxy = true
-        proxy_balance = vault.getBalance()
-    }
+    //let ref = acct.capabilities.borrow<&{FlowColdStorageProxy.Vault}>(FlowColdStorageProxy.VaultCapabilityPublicPath)
+    //if let vault = ref {
+    //    is_proxy = true
+    //    proxy_balance = vault.getBalance()
+    //}
```

Comment:
Not sure if this is supposed to be commented out. If so, also updating the statically declared capability type, similar to the suggestion in the go script below
```suggestion
    let ref = acct.capabilities.borrow<&FlowColdStorageProxy.Vault>(FlowColdStorageProxy.VaultCapabilityPublicPath)
    if let vault = ref {
        is_proxy = true
        proxy_balance = vault.getBalance()
    }
```